tox.ini

```diff
@@ -10,3 +10,6 @@ commands=
 [pytest]
 addopts = --pep8 --cov straitlets --cov-report term-missing --cov-report html
 testpaths = straitlets
+filterwarnings =
+    # PyYAML==3.13
+    ignore:Using or importing the ABCs:DeprecationWarning:yaml.constructor:126
```

Comment:
Only one warning, from PyYAML. Full output:

```
straitlets/ext/tests/test_click.py::test_yaml_file
  site-packages/yaml/constructor.py:126: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
    if not isinstance(key, collections.Hashable):
```

We require `pyyaml>=3.11`, so it should go away once it's fixed upstream. There's a couple open PRs with a fix already.

Apparently there is a Situation with PyYAML releases that we might want to keep an eye on, though: see https://github.com/yaml/pyyaml/issues/193, https://github.com/yaml/pyyaml/pull/194.